tests-e2e/cypress/integration/runs/rdp_main_header_spec.js

```diff
@@ -395,6 +395,31 @@ describe('runs > run details page > header', () => {
                     cy.findByRole('dialog', {name: /Run Actions/i}).should('not.exist');
                 });
             });
+
+            describe('leave run', () => {
+                it('can leave run', () => {
+                    // # Add viewer user to the channel
+                    cy.apiAddUserToChannel(playbookRun.channel_id, testViewerUser.id);
+
+                    // # Change the owner to testViewerUser
+                    cy.apiChangePlaybookRunOwner(playbookRun.id, testViewerUser.id);
+                    cy.wait(500);
+
+                    // # Click on leave run
+                    getDropdownItemByText('Leave and unfollow run').click();
+
+                    // # confirm modal
+                    cy.get('#confirmModal').get('#confirmModalButton').click();
+
+                    // * Assert that the Participate button is shown
+                    getHeader().findByText('Participate').should('exist');
```

Comment:
would you want to additionally check that is visible in the screen?
```suggestion
                    getHeader().findByText('Participate').should('be.visible');
```